docs/usage/golang.md

```diff
@@ -15,8 +15,9 @@ Renovate supports upgrading dependencies in `go.mod` files and their accompanyin
 1. If Renovate finds an update, Renovate will update `go.mod` to the new value
 1. Renovate runs `go get` to update the `go.sum` files
 1. If the user has enabled the option `gomodUpdateImportPaths` in the [`postUpdateOptions`](https://docs.renovatebot.com/configuration-options/#postupdateoptions) array, then Renovate uses [mod](https://github.com/marwan-at-work/mod) to update import paths on major updates, which can update any Go source file
-1. If the user has enabled the option `gomodTidy` or `gomodTidy1.17` in the [`postUpdateOptions`](https://docs.renovatebot.com/configuration-options/#postupdateoptions) array, then Renovate runs `go mod tidy` or `go mod tidy -compat=1.17` respectively, which itself can update `go.mod` and `go.sum`.
+1. If the user has enabled the option `gomodTidy`, `gomodTidy1.17` `gomodTidyE` in the [`postUpdateOptions`](https://docs.renovatebot.com/configuration-options/#postupdateoptions) array, then Renovate runs `go mod tidy`, `go mod tidy -compat=1.17` or `go mod tidy -e` respectively, which itself can update `go.mod` and `go.sum`.
```

Comment:
```suggestion
1. If the user has enabled the option `gomodTidy`, `gomodTidy1.17` or `gomodTidyE` in the [`postUpdateOptions`](https://docs.renovatebot.com/configuration-options/#postupdateoptions) array, then Renovate runs `go mod tidy`, `go mod tidy -compat=1.17` or `go mod tidy -e` respectively, which itself can update `go.mod` and `go.sum`.
```

Reply:
```suggestion
1. If the user has enabled the option `gomodTidy`, `gomodTidy1.17` or `gomodTidyE` in the [`postUpdateOptions`](https://docs.renovatebot.com/configuration-options/#postupdateoptions) array, then Renovate runs `go mod tidy`, `go mod tidy -compat=1.17` or `go mod tidy -e` respectively, which itself can update `go.mod` and `go.sum`
```

We don't use punctuation at the end of list items.